index.css

```diff
@@ -0,0 +1,3 @@
+body {
+  color: black;
```

Comment:
[ ] kindly use a CSS class just for your header, not for the whole body. it's on the project requirements.🤓
![css](https://user-images.githubusercontent.com/62156607/204639392-0f362964-1086-448e-9038-b9012c62fd4b.PNG)